payment/src/main/java/org/killbill/billing/payment/api/DefaultApiBase.java

```diff
@@ -79,6 +80,13 @@ protected void logAPICall(final String transactionType, final Account account, f
                 logLine.append(", currency = ")
                        .append(currency);
             }
+
+            final String requestId = Request.getPerThreadRequestData() != null
```

Comment:
Let's actually get rid of all requestId logging (here and below). I've opened https://github.com/killbill/killbill/issues/409 for tracking (we should really use the user token instead, which follows bus events and notifications).